src/components/TheNavigation.vue

```diff
@@ -1,113 +1,160 @@
+        @mouseover="isDesktopResolution ? open(menu.name) : null"
+        @mouseleave="isDesktopResolution ? closeAll() : null">
+        <menu-item :menu="menu"/>
       </li>
     </ul>
-  </div>
+  </nav>
 </template>
 
 <script setup>
-import AppLink from '@/components/AppLink'
-import ComingSoonTooltip from '@/components/ComingSoonTooltip'
-</script>
+import { isDesktop } from '@/utils/screen'
 
-<style scoped>
-.navigation {
-  display: flex;
-  flex-direction: column;
+const menuOptions = ref([{
+  name: 'Blockchain',
+  isActive: false,
+  submenu: [
+    {
+      name: 'Transactions',
+      path: '/transactions',
+    },
+    {
+      name: 'Names',
+      path: '/names',
+    },
+    {
+      name: 'Smart Contracts',
+      path: '/contracts',
+    },
+    {
+      name: 'Oracles',
+      path: '/oracles',
+    },
+    {
+      name: 'State Channels',
+      path: '/state-channels',
+    },
+    {
+      name: 'Hyperchains',
+      path: '/hyperchains',
+      isDisabled: true,
+    },
+  ],
+},
+{
+  name: 'Tokens',
+  isActive: false,
+  submenu: [
+    {
+      name: 'AEX9 Tokens',
+      path: '/tokens',
+    },
+    {
+      name: 'AEX141 Tokens (NFTs)',
+      path: '/nfts',
+    },
+    {
+      name: 'DEX Trades',
+      path: '/trades',
+      isDisabled: true,
+    },
+  ],
+},
+{
+  name: 'Analytics',
+  isActive: false,
+  submenu: [
+    {
+      name: 'Top Accounts',
+      path: '/top-accounts',
+      isDisabled: true,
+    },
+    {
+      name: 'Charts',
+      path: '/charts',
+      isDisabled: true,
+    },
+  ],
+},
+{
+  name: 'Developers',
+  isActive: false,
+  submenu: [
+    {
+      name: 'Contract Verification',
+      path: '/verification',
+      isDisabled: true,
+    },
+  ],
+}])
 
-  @media (--desktop) {
-    gap: var(--space-5);
-    flex-direction: row;
-  }
+function open(name) {
+  menuOptions.value.find(item => item.name === name).isActive = true
+}
 
-  &__item {
-    text-align: center;
+const activeItemName = computed(() => {
+  const activeItem = menuOptions.value.find(item => item.isActive)
+  return activeItem ? activeItem.name : null
+})
+
+function toggle(name) {
+  if (activeItemName.value !== name) {
+    closeAll()
+    open(name)
+  } else {
+    closeAll()
   }
+}
 
-  &__link {
-    color: var(--color-midnight);
-    display: block;
-    padding: var(--space-3) 0;
-    font-size: 16px;
-    line-height: 24px;
-    letter-spacing: 0.002em;
+function closeAll() {
+  menuOptions.value.forEach(item => {
+    item.isActive = false
+  })
+}
 
-    @media (--desktop) {
-      padding: 0;
-    }
+onMounted(() => {
+  updateIsDesktopResolution()
+  window.addEventListener('resize', updateIsDesktopResolution)
+})
 
-    &:hover {
-      text-decoration: underline;
-    }
+onBeforeUnmount(() => {
+  window.removeEventListener('resize', updateIsDesktopResolution)
+})
+
+const isDesktopResolution = ref(null)
+
+function updateIsDesktopResolution() {
+  isDesktopResolution.value = isDesktop()
```

Comment:
Will follow up with this
https://github.com/aeternity/aescan/issues/105
We will reuse it here and also another places